spec/tests/openstudio_meta_spec.rb

```diff
@@ -68,14 +68,30 @@ class OpenStudioMeta
 class LocalRspecTest
 end
 
+# Set obvious paths for start-local & run-analysis invocation
 mongod_exe = which('mongod')
 ruby_cmd = 'ruby'
 meta_cli = File.absolute_path(File.join(File.dirname(__FILE__), '../../bin/openstudio_meta'))
 project = File.absolute_path(File.join(File.dirname(__FILE__), '../files/'))
 server_rspec_test_dir = File.absolute_path(File.join(File.dirname(__FILE__), '../unit-test/'))
+
+# Attempt to locate oscli if it is not set via env var for the rspec test
+if ENV['OPENSTUDIO_TEST_EXE']
+  unless File.exist? ENV['OPENSTUDIO_TEST_EXE']
+    raise "Can't find OPENSTUDIO_TEST_EXE at #{ENV['OPENSTUDIO_TEST_EXE']}"
+  end
+else
+  oscli_path = which('openstudio')
+  if oscli_path
+    ENV['OPENSTUDIO_TEST_EXE'] = oscli_path
+    else
+      raise "Can't find openstudio cli on path - please specify via env var OPENSTUDIO_TEST_EXE"
+  end
+end
+
 # remove leftover files from previous tests if they exist
 to_rm = [File.join(project, 'temp_data'), File.join(project, 'localResults')]
-to_rm.each { |dir| FileUtils.rm_rf(dir) if Dir.exist? dir }
+# to_rm.each { |dir| FileUtils.rm_rf(dir) if Dir.exist? dir }
```

Comment:
did you forget to uncomment this or is this on purpose?

Reply:
presumably this was added for a reason, but debugging is muuuch easier when we don't clean up the temp_data directory.  i'll leave commented but add a comment above.